src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const trimmedStyles = sourceString.split(';').map((style) => style.trim());
```

Comment:
I think u can do it in middle of the `reduce`. Then u won't have to iterate over the string twice, but it's `Up to u`